assets/src/pages/room/components/StreamPlayer/ScreenSharingPlayers.tsx

```diff
@@ -17,7 +17,8 @@ type Props = {
 
 const ScreenSharingPlayers: FC<Props> = ({ streams }: Props) => {
   return (
-    <div className="h-full mb-3 md:mr-3 md:mb-none active-screensharing-grid grid-cols-1 md:grid-cols-1">
+    //TODO remove max-h
+    <div className="h-full mb-3 md:mr-3 md:mb-none active-screensharing-grid grid-cols-1 md:grid-cols-1 max-h-[32rem]">
```

Comment:
What does this TODO mean?

Reply:
This is a temporary solution that prevents page overflow. This should be removed when a new grid is introduced.

Reply:
I'd include that in the comment
```suggestion
    //TODO remove max-h when a new grid is introduced.
    <div className="h-full mb-3 md:mr-3 md:mb-none active-screensharing-grid grid-cols-1 md:grid-cols-1 max-h-[32rem]">
```